Polyfills/XMLHttpRequest/Source/XMLHttpRequest.cpp

```diff
@@ -77,12 +80,16 @@ namespace Babylon::Polyfills::Internal
                 InstanceAccessor("responseType", &XMLHttpRequest::GetResponseType, &XMLHttpRequest::SetResponseType),
                 InstanceAccessor("responseURL", &XMLHttpRequest::GetResponseURL, nullptr),
                 InstanceAccessor("status", &XMLHttpRequest::GetStatus, nullptr),
+                InstanceAccessor("ok", &XMLHttpRequest::Ok, nullptr),
+                InstanceAccessor("headers", &XMLHttpRequest::GetAllResponseHeaders, nullptr),
```

Comment:
I'm confused by this. The spec is a function call [getAllResponseHeaders](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/getAllResponseHeaders). It's not a property?